_docs/services/relational-database.md

```diff
@@ -109,6 +124,24 @@ If the response is `OK` instead of `FAILED` then your database is ready to use.
 
 The cloud.gov team aims to provide clearer status indicators in a future release of our service broker.
 
+## Update an instance
+
+To update an existing service instance run the following command:
+
+```sh
+cf update-service aws-rds ${SERVICE_NAME} -p ${SERVICE_PLAN_NAME}
+```
+
+There are several caveats regarding this command:
+
+- You can only update dedicated RDS instances; updates to shared instances are not supported.
+- You can only update using plans with the same database engine as your existing service instance. This means that if your original service instance was using a PostgreSQL plan (e.g., `micro-psql`), you can only update to one of the other `psql`-based plans.
+- You can **only** switch service plans at the moment; you cannot do things like update your database size at this time.
```

Comment:
Do you mean `you cannot do things like update your database size at a specified time`? Not really sure what this sentence means.

Reply:
Thanks for calling this out - it is meant to say that you can only switch instance classes at this time, nothing else.  You cannot change the database size, you can't adjust any other parameters, etc.